src/components/pre-footer/PreFooterMid.tsx

```diff
@@ -9,7 +9,7 @@ const PreFooterMid = () => {
       <ResponsiveRow breakpoint="lg">
         <BoundedContainer breakpoint="lg" width="60%" margin="">
           <SubHeading>
-            Are you an end user and adopted LitmusChaos{" "}
+            Are you an end user and adopted{" "}
```

Comment:
`Are you an end user and adopted?`
That makes a weird sentence...please get it reviewed.
This might be `Are you an end-user or an adopter?`

Reply:
you don't have to make this exact change...it's just an example

Reply:
Okay, will confirm. This was the exact text given to us.